web-common/src/features/dashboards/time-dimension-details/TimeDimensionDetailsTable.svelte

```diff
@@ -0,0 +1,45 @@
+<script lang="ts">
+  import { flexRender } from "@tanstack/svelte-table";
+  import Pivot from "./Pivot.svelte";
+  import TimeDimensionDetailsTableCell from "./TimeDimensionDetailsTableCell.svelte";
+
+  // Mock state for now
+  let state = {
+    getRowSize: () => 35,
```

Comment:
We should consider passing all this as context and follow the line chart design. Where a top level wrapper component allows for configuration and sets a context. All the components down the tree uses this. Perhaps when we actually use this and start generalising it a bit.

Reply:
For now, this component is just a mock of how we will set up state.
A Pivot context could work, but at this stage feels like overkill. We only prop drill this property 1 level. We definitely can revisit it later if we find a real need for it.